frontend/packages/core/src/Input/toggle-button-group.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import {
+  ToggleButton as MuiToggleButton,
+  ToggleButtonGroup as MuiToggleButtonGroup,
+} from "@material-ui/lab";
+
+export interface ToggleButtonGroupProps {
+  /** A boolean to decide if the user can select multiple options
+   * simultaneously. If true, users will be able to select multiple options,
+   * whereas if false, an exclusive choice will be enforced.
+   */
+  multiple?: boolean;
+  /** The current value that is selected in the group. Usually this will be
+   * a useState value.
+   */
+  currentValue: string;
```

Comment:
can you pick this from the material ui toggle button group `value` prop instead of using a custom `currentValue` prop?